test/samples/runtime-helpers-commonjs/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": [ "@babel/env" ],
+  "plugins": [ "@babel/external-helpers", "@babel/transform-runtime" ]
```

Comment:
are external helpers needed for this test? without testing (😉) it seems to me that the issue becomes relevant when using transform-runtime and some module transform, so I suspect external-helpers are not needed here

Reply:
It was actually copied on "runtime-helpers" test. That's the only reason. I think it make sense to keep it just to make sure that this is the only difference between the two.

Reply:
If it doesnt matter for the test I would prefer keeping it minimal